src/App.vue

```diff
@@ -1,15 +1,15 @@
 <template>
-    <div class="w-full min-h-screen px-1vw reset" :class="[theme]">
-        <NavBar />
-
+    <NavBar />
```

Comment:
The Vue compiler is complaining here because `template` elements should only have one direct child. You should probably wrap all the content here into a `div` to make Vue happy :)

Reply:
But that shouldn't be the case in Vue 3 😕

Reply:
Ouch, maybe I'm using an outdated VSCode extension version :/